client/src/components/MultiJurisdictionAudit/RoundManagement/generateSheets.ts

```diff
@@ -105,8 +105,16 @@ export const downloadDataEntry = (auditBoards: IAuditBoard[]): void => {
         20,
         140
       )
+    } else if (board.currentRoundStatus.numSampledBallots === 0) {
+      auditBoardsWithoutBallots.push(board.name)
     }
   })
+  if (auditBoardsWithoutBallots.length) {
+    auditBoardCreds.addPage('letter')
+    auditBoardsWithoutBallots.forEach((name, i) => {
+      auditBoardCreds.text(`${name}: No ballots`, 20, i * 10 + 20)
+    })
+  }
```

Comment:
do you think there's any easy way to test the pdf generation? like maybe just an integration test where we download the pdf and click the link

Reply:
@jonahkagan I've been digging into that. If you have any ideas, I have a thread on slack about it. I haven't found any easy ways yet, though. Or any hard ways either, for that matter. :stuck_out_tongue_closed_eyes: